.github/workflows/pr-audit.yml

```diff
@@ -0,0 +1,33 @@
+name: Pull Request Security Audit
+
+on:
+  push:
+    paths:
+      - '**/**/Cargo.toml'
+  pull_request:
+    paths:
+      - '**/**/Cargo.toml'
+
+jobs:
+  security-audit:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'audit skip')"
```

Comment:
Is it better to change to `if: "contains(github.event.head_commit.message, 'audit')"`?

Reply:
or check `github.event.comment.body` when pr is created?